### PR TITLE
[4.1] tag active focus

### PR DESCRIPTION
**Pull request by brianteeman**
Pull Request for Issue #37864  .

### Summary of Changes
add a margin around the tags so that the focus ring is displayed correctly

### Testing Instructions
This is a css change so either use a pre-built package or npm run build:css

### Actual result BEFORE applying this Pull Request
image

### Expected result AFTER applying this Pull Request

image

### Documentation Changes Required

n/a

### CODE REVIEW

**Comment by chmst** [comment]
I have tested this item :white_check_mark: successfully on 3fa7b268d83b57da1a4239c397952fce90622317<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37877">issues.joomla.org/tracker/joomla-cms/37877</a>.</sub>

**Comment by Kostelano** [comment]
I have tested this item :white_check_mark: successfully on 3fa7b268d83b57da1a4239c397952fce90622317<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37877">issues.joomla.org/tracker/joomla-cms/37877</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37877">issues.joomla.org/tracker/joomla-cms/37877</a>.</sub>

**Comment by richard67** [comment]
@brianteeman After I had updated your branch to make the npm step pass in drone, the scss code style check fails in drone: https://ci.joomla.org/joomla/joomla-cms/54621/1/24 .

File: build/media_source/templates/site/cassiopeia/scss/blocks/_tags.scss [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 .tags {
   .list-inline-item {
-    margin-bottom: .5rem;
+    margin: .25rem .25rem .5rem .25rem;
```
Comment (richard67) [inline build/media_source/templates/site/cassiopeia/scss/blocks/_tags.scss]:
```suggestion
    margin: .25rem .25rem .5rem;
```
That’s what the linter complains about in drone. It says unexpected long hand value where short hand value is expected.

**Comment by brianteeman** [comment]
Thanks for fixing drone. I thought it might have been better as short hand but couldnt remember if we just left that to the compiler..

**Comment by sandewt** [comment]
Thanks

**Comment by Quy** [comment]
Thanks

**Pull request merged**